topic_folders/instructor_training/duties_agreement.md

```diff
@@ -12,31 +12,39 @@ discussions about Instructor training. Trainers have opportunities to influence
 community, through leadership roles. Trainers often play leadership roles in their home communities, mentoring local Instructors, and modeling best practices.
 
 
-##### Certification Renewal Process
+##### Active Status Renewal Process
 
-Version 1.0.0 -- Approved 16 February, 2022
+Version 1.0.1 -- Draft
```

Comment:
will we put the date in here when approved?

Reply:
```suggestion
Version 1.0.1 -- Approved 8 February, 2023
```